docs/docs/dev_docs/contracts/main.md

```diff
@@ -2,7 +2,7 @@
 
 ## What is Aztec.nr?
 
-**Aztec.nr** is a library for writing Aztec smart contracts.
+**Aztec.nr** is a framework for writing Aztec smart contracts.
```

Comment:
Interesting. I am good with this distinction

Reply:
In the state_model.md doc, it says "The goal of the Aztec.nr smart contract library" -- should that say framework or am I missing the distinction?